tests/gptq/test_gptq.py

```diff
@@ -0,0 +1,145 @@
+# coding=utf-8
```

Comment:
Let's put all tests in a `quantization` folder instead of creating one folder per quantization method.

Reply:
done